### PR TITLE
Support ignoring specified forms when checking for CSRF vulnerabilities

### PR DESCRIPTION
**Pull request by psiinon**

### CODE REVIEW

**Comment by kingthorin** [comment]
Is there a help update that goes with this? :smile: Otherwise looks good.

**Comment by psiinon** [comment]
The help for the new param is included in the Messages.properties file - it is shown when you edit it.
I've also added help to the relevant rules that use it, via https://github.com/zaproxy/zap-extensions/pull/462
Do we need any more help? I didnt think so...

**Comment by kingthorin** [comment]
Just wondered if we'd have a help page on the dialog and configuring things. Guess that's a `zap-core-help` update and might be applicable to your earlier change related to timing settings as well...

**Comment by psiinon** [comment]
Yeah, I need to push the changes to add help for the Options/ Rule Config panel. This wasnt going to include any help for the params as thats included with them. That ok?

File: src/org/zaproxy/zap/extension/ruleconfig/DialogEditRuleConfig.java [hunk 1]
```diff
@@ -61,7 +61,9 @@ public void init (RuleConfig rc, RuleConfigTableModel model) {
             desc = Constant.messages.getString(rc.getKey());
         }
         this.addMultilineField(FIELD_DESC, desc);
-        ((ZapTextArea)this.getField(FIELD_DESC)).setEditable(false);
+        ZapTextArea descField = ((ZapTextArea)this.getField(FIELD_DESC));
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/ruleconfig/DialogEditRuleConfig.java]: The surrounding parentheses could be removed.

**Comment by thc202** [comment]
That's fine with me.

**Comment by psiinon** [comment]
Removed the spurious brackets

**Comment by kingthorin** [comment]
Sounds good.

**Comment by psiinon** [comment]
this ok to merge now?

**Pull request merged**